### DISCUSSION TITLE
What is the Site URL I should specify ?

### DISCUSSION THREAD

**Discussion by accountsquality** [supabase/supabase#26708] [Questions]
Hello, so I am working on a mobile app and for now, I do not want anybody to sign up manually. I will add users manually from "authentication" on supabase.

For the site URL, since it is currently set to localhost:3000, what is the "correct" URL in this case ? where should the email confirmation redirect the user to ?

**Comment by GaryAustin1** [comment]
This has a bunch of info on different siteURLs to use for different environments. https://supabase.com/docs/guides/auth/redirect-urls
